scripts/examples/mixed_with_len_func/str_comp_n_len2.py

```diff
@@ -0,0 +1,5 @@
+"""Tests behaviour based on multiple disjunctive conditions involving string equality and length checks."""
+def strComp_n_len4(param1: str, param2: str, param3: str) -> bool:    
+    if (param1 == "Prime number" or len(param2) == 13) or param3 != "Fibonacci" or len(param3) != 7:        
```

Comment:
The conditions `param3 != "Fibonacci" or len(param3) != 7` are too easy and Pynguin will have no trouble because it just needs param3 to be any string with length other than 7. Idea: Change these to:

`(param2 == "Fibonacci" and len(param3) >= 7)`

You can also make it nested and add some more conditions in a second if-statement.

Reply:
fixed